src/main/java/com/jmb/springfactory/model/bo/ConnectedUser.java

```diff
@@ -0,0 +1,24 @@
+package com.jmb.springfactory.model.bo;
+
+import java.util.Collection;
+
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.User;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class ConnectedUser extends User {
+
+    private static final long serialVersionUID = -437685269384982011L;
+
+    private String token;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "token" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)